src/Stan/Report/Html.hs

```diff
@@ -133,6 +139,38 @@ stanAnalysis (analysisToNumbers -> AnalysisNumbers{..}) =
         )
     )
 
+stanSummary analysis AnalysisNumbers{..} =
+    ( divClass "row" (blockP "Summary of the static analysis report")
+    # ul_
+        ( li_ (strong_ "Project health: " # prettyHealth anHealth)
+        # li_ ("The project " # showProjectHealth (toProjectHealth anHealth))
+        # summary
+        )
+    )
+  where
+    showProjectHealth :: ProjectHealth -> Text
+    showProjectHealth = toText . \case
+        Unhealthy    -> "is unhealthy"
+        LowHealth    -> "has low health"
+        MediumHealth -> "has medium health"
+        Healthy      -> "is healthy"
+
+    summary = case createSummary analysis of
+        Nothing -> Left $ li_ "Congratulations! Your project has zero vulnerabilities!"
```

Comment:
Maybe "well done" or "good job" fits here better? It is their achievement, and we can highlight that, what do you think?